demos/object_detection_demo/cpp/main.cpp

```diff
@@ -315,12 +318,20 @@ int main(int argc, char *argv[]) {
         bool keepRunning = true;
         int64_t frameNum = -1;
         std::unique_ptr<ResultBase> result;
+        uint32_t framesProcessed = 0;
+        cv::VideoWriter videoWriter;
 
         while (keepRunning) {
             if (pipeline.isReadyToProcess()) {
                 //--- Capturing frame
                 auto startTime = std::chrono::steady_clock::now();
                 curr_frame = cap->read();
+                if (frameNum == -1) {
+                    if (!FLAGS_o.empty() && !videoWriter.open(FLAGS_o, cv::VideoWriter::fourcc('I', 'Y', 'U', 'V'),
```

Comment:
Why `'I', 'Y', 'U', 'V'`? For python it's `'MJPG'`.

Reply:
Changed.